packages/mkaudio/mkaudio.1.1.0/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+author: "John Else <john.else@gmail.com>"
+maintainer: "John Else <john.else@gmail.com>"
+homepage: "https://github.com/johnelse/mkaudio"
+bug-reports: "https://github.com/johnelse/mkaudio/issues"
+dev-repo: "git://github.com/johnelse/mkaudio"
+build: [
+  ["dune" "build" "@install" "-p" name]
+]
+build-test: [
+  ["dune" "runtest" "-p" name]
+]
+install: [
```

Comment:
This part in unnecessary. Please have a look at other packages using dune for more examples of how it should be done.

For more informations here is the documentation: https://dune.readthedocs.io/en/latest/usage.html#invocation-from-opam

Reply:
Thanks, updated. I've left out the `opam subst` line for now, since there's nothing to substitute in the code at the moment.